src/Promitor.Integrations.Sinks.Prometheus/Collectors/PrometheusSystemMetricsSink.cs

```diff
@@ -36,5 +36,23 @@ public Task WriteGaugeMeasurementAsync(string name, string description, double v
 
             return Task.CompletedTask;
         }
+        
+        /// <summary>
+        ///     Records measurement for a histogram instrument 
+        /// </summary>
+        /// <param name="name">Name of the metric</param>
+        /// <param name="description">Description of the metric</param>
+        /// <param name="value">New measured value</param>
+        /// <param name="labels">Labels that are applicable for this measurement</param>
+        /// <param name="includeTimestamp">Indication whether or not a timestamp should be reported</param>
+        public Task WriteHistogramMeasurementAsync(string name, string description, double value, Dictionary<string, string> labels, bool includeTimestamp)
+        {
+            var orderedLabels = labels.OrderByDescending(kvp => kvp.Key).ToDictionary(kvp => kvp.Key, kvp => kvp.Value);
+
+            // TODO: are histogram instruments created on every invocation? Would that interfere with correctness? 
```

Comment:
Is this still relevant?

Reply:
Well, I was worried if this would create a new instrument and reset histogram counts every time, but it didn't seem to be the case. I saw histogram values accumulate properly when I ran this in our staging environment

Reply:
So no it's no longer relevant. I removed it